### ISSUE TITLE
[D-med] Warning screen reader users that filters have been updated

### ISSUE THREAD

**Issue by daniel-montalvo** [w3c/wcag-em-report-tool#320]
Currently  when one checks and unchecks filters in step 4, page content is dynamically updated (success criteria are added or removed). An aria-live region warning screen reader users of that would be appreciated. Extend this to other places where it might be applicable.

Not blocking publication.

**Comment by nitedog** [comment]
Visually, the only (real) indication of success is the state of the button, which I think is conveyed to screen reader users (please correct me if I'm wrong on this). If that indication is not sufficient, then it probably needs improvement for all users, not just for screen reader users.

**Comment by daniel-montalvo** [comment]
> 
> 
> Visually, the only (real) indication of success is the state of the button, which I think is conveyed to screen reader users (please correct me if I'm wrong on this).

Yes, you can tell whether the checkbox is checked or unchecked. What a screen reader user cannot easily guess is what happens after you have checked or unchecked the checkbox. Users with vision can see at a glance how Success Criteria are added or removed from the screen as they play with the checkboxes, right? Screen reader users need to read the paragraph under the "Info" button to understand the functionality. Otherwise, thay have to go down the page to determine the remaining success criteria after they have customized the filters. I can imagine somebody who comes across this tool for the first time might not be aware of what it implies to check or uncheck those checkboxes, since they might skip the "Info" button.

> If that indication is not sufficient, then it probably needs improvement for all users, not just for screen reader users.

That would be another way to approach this issue. Making the sentences under the "Info" button more prominent would help everybody (including screen reader users) understand the functionality.

**Comment by nitedog** [comment]
> Users with vision can see at a glance how Success Criteria are added or removed from the screen as they play with the checkboxes, right?

Not really because the changes happen far down (you have to scroll)
